classes/Shirt.java

```diff
@@ -0,0 +1,12 @@
+package classes;
+
+public class Shirt {
+
+    public int sizeOfShirt;
+    public String colorOfShirt;
```

Comment:
```suggestion
    public String color;
```
you are in shirt class so no need shirt in var name. we want them informative but short too